P3826/P3826.md

```diff
@@ -1471,13 +1470,13 @@ follows:]{.ednote}
 >         pack of lvalue subexpressions designating objects decay-copied from `vals`. Then
 >
 >         - [10.2.1.1]{.pnum} If `bulk_sndr` is the result of calling `bulk_chunked(child,
->           policy, shape, f)`, `sch_->schedule_bulk_chunked(shape, r, s)`{.cpp} is
+>           policy, shape, f)`, ``@_`sch_`_@->schedule_bulk_chunked(shape, r, s)``{.cpp} is
```

Comment:
i think this needs triple ticks, right? 
```suggestion
>           policy, shape, f)`, ```@_`sch_`_@->schedule_bulk_chunked(shape, r, s)```{.cpp} is
```